src/basics.php

```diff
@@ -51,7 +51,13 @@ function debug($var, $showHtml = null, $showFrom = true)
         $line = '';
         $lineInfo = '';
         if ($showFrom) {
-            $trace = Debugger::trace(['start' => 1, 'depth' => 2, 'format' => 'array']);
+            $trace = Debugger::trace(['start' => 1, 'depth' => 3, 'format' => 'array']);
+            if (count($trace) > 1
+                && $trace[1]['function'] === 'dd'
```

Comment:
>  Having a method check for another method (dd) which calls it inside a method (debug), that doesn't sound good.

Normally yes but this is just a debugging tool and when dealing which stack traces such specific checks are often required. Having a consistent display for developer is more important here IMO.

Only other option I see is moving the location display code into a separate function which can be used by both debug() and dd() but adding another global function would be even less ideal.